tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -324,6 +332,7 @@ def test_input_padding(
     ],
 )
 @pytest.mark.skip_on_windows
```

Comment:
I opted to keep it because I'm not sure why it's there in the first place. Do you know? Figured if Windows user runs it locally it should maybe still be skipped? Not sure

Reply:
`transformers` is runnable on windows too, so I don't see why they should be skipped. We should remove it and see what happens maybe?